.github/workflows/integration.yaml

```diff
@@ -6,14 +6,15 @@ env:
 jobs:
   test:
     name: Test
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        python-version: [3.7]
-        subset: ['upload_datatype', 'extended_metadata', 'kubernetes', 'not (upload_datatype or extended_metadata or kubernetes)']
+        python-version: ['3.7']
+        subset: ['kubernetes']
```

Comment:
I suppose this change was just for testing?

Reply:
Haha, yeah, one sec